source/administration/object-management/data-compression.rst

```diff
@@ -48,7 +48,8 @@ You can control which objects are compressed by specifying the desired file exte
 Excluded File Types
 ~~~~~~~~~~~~~~~~~~~
 
-Some data cannot be effectively compressed, such as video or already compressed data.
+Some data cannot be effectively compressed.
+For example: video, already compressed data, or very small files.
```

Comment:
```suggestion
For example: video, already compressed data, or files less than 4KiB.
```

Reply:
👍🏻 will add manually to reduce merge nonsense